podman/containers/__init__.py

```diff
@@ -0,0 +1,315 @@
+"""images provides the operations against images for a Podman service."""
+import json
+import logging
+
+import podman.errors as errors
+
+
+def make_call(api, endpoint, method='GET', params=None, body=None):
+    """A helper function to keep things DRY"""
+    path = api.join(endpoint, params)
+    print(path)
+    urlify = api.quote(path)
+    response = api.request(method, urlify)
```

Comment:
body is unused? It seems like we should be using body in combination with {'Content-type': 'application/x-www-form-urlencoded'} to send the parameters when method='POST'

Reply:
Fair point, it has been so long since I looked at this code I'm not quite sure why that is...  Fixed based on your subsequent comments.